test/utils/ReentrancyGuard.test.js

```diff
@@ -9,28 +9,26 @@ require('chai')
   .should();
 
 contract('ReentrancyGuard', function () {
-  let reentrancyMock;
-
   beforeEach(async function () {
-    reentrancyMock = await ReentrancyMock.new();
-    const initialCounter = await reentrancyMock.counter();
-    initialCounter.should.be.bignumber.equal(0);
+    this.reentrancyMock = await ReentrancyMock.new();
+    this.initialCounter = await this.reentrancyMock.counter();
```

Comment:
This should not be added to `this` :stuck_out_tongue: 

In fact, I seemed to have missed this line in #1216. It should be replaced with:
```
(await this.reentrancyMock.counter()).should.be.bignumber.equal(0);
```

Reply:
That changes were done just before my break. :smile: